bulletroboy/forces_mapper.py

```diff
@@ -0,0 +1,156 @@
+import rclpy
+from rclpy.node import Node
+
+# from sensor_msgs.msg import JointState
+from roboy_simulation_msgs.msg import Collision
+import yaml
+from roboy_simulation_msgs.srv import LinkIdFromName
+from roboy_simulation_msgs.srv import LinkNameFromId
+from asgiref.sync import async_to_sync
+
+class ForcesMapper(Node):
+    def __init__(self):
+        super().__init__('forces_imitator')
+
+        self.result = None
```

Comment:
Maybe use a different name that is more intuitive so that we know it refers to operator_collision.

Reply:
see my comment in `transformFromRobotToOperator` on this topic